### PR TITLE
Adding docker_container

### PR DESCRIPTION
**Pull request by chouseknecht**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

docker_container.py 
##### ANSIBLE VERSION

```
ansible 2.1.0 (docker_testing 9bb069f873) last updated 2016/04/18 12:36:03 (GMT -400)
  lib/ansible/modules/core: (detached HEAD cf01087a30) last updated 2016/04/05 16:31:53 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD 204b4bab56) last updated 2016/04/05 16:31:56 (GMT -400)
  config file =
  configured module search path = Default w/o overrides
```

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @chouseknecht for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by bcoca** [comment]
hmm, there is no diff??

**Comment by chouseknecht** [comment]
@bcoca what's holding this up?

When I look at the diff I see this:

```
1,625 additions, 0 deletions not shown because the diff is too large. Please use a local Git client to view these changes.
```

Not sure I can do anything to fix that.

**Comment by bcoca** [comment]
i don't know what is wrong, but there is nothing to review, you might need to resubmit a clean branch

**Comment by sivel** [comment]
I think github is just saying that the file is too large to be shown in the diff view.  Due to the huge size of the file though, it makes it hard to comment on the PR.

The full file can be seen at https://github.com/chouseknecht/ansible-modules-core/blob/docker_container/cloud/docker/docker_container.py

**Comment by jctanner** [comment]
shipit

File: cloud/docker/docker_container.py [hunk 1]
```diff
@@ -0,0 +1,1625 @@
...
+            oom_killer=host_config.get('OomKillDisable'),
+            pid_mode=host_config.get('PidMode'),
+            privileged=host_config.get('Privileged'),
+            expected_ports=host_config.get('PortBindings'),
+            read_only=host_config.get('ReadonlyRootfs'),
+            restart_policy=restart_policy.get('Name'),
+            restart_retries=restart_policy.get('MaximumRetryCount'),
+            # Cannot test shm_size, as shm_size is not included in container inspection results.
+            # shm_size=host_config.get('ShmSize'),
+            security_opts=host_config.get("SecuriytOpt"),
```
Comment (jpcope) [inline cloud/docker/docker_container.py:1018]: typo?

Most likely should have been "SecurityOpt" and not "SecuriytOpt".

**Pull request merged**